gluten-substrait/src/main/scala/org/apache/gluten/extension/columnar/validator/Validators.scala

```diff
@@ -116,7 +116,7 @@ object Validators {
     }
   }
 
-  private class FallbackComplexExpressions(threshold: Int) extends Validator {
+  class FallbackComplexExpressions(threshold: Int) extends Validator {
```

Comment:
Instead of making this public, could move the shared code into `ExpressionUtils`